plugins/redshift/dbt/include/redshift/sample_profiles.yml

```
@@ -0,0 +1,25 @@
+default:
+  outputs:
+
+    dev:
+      type: redshift
+      threads: [1 or more]
+      host: [host]
+      port: 5439
```

I think, if you try hard enough, it's possible to make this a diff number (and different if you're connecting via SSH)
```suggestion
      port: [port]
```